src/components/Form/Form.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import './Form.css';
+
+const Form = (props) => {
+  const [currentQuestion, setCurrentQuestion] = useState(1);
+  const [name, setName] = useState('');
+  const [problemType, setProblemType] = useState('');
+  const [techType, setTechType] = useState('');
+  const [vent, setVent] = useState('');
+  const [ventTwo, setVentTwo] = useState('');
+
+  return (
+    <form>
+      {currentQuestion === 1 && <label>
+      What can I call you?
+        <input
+          aria-label="name input"
+          className="name-input"
+          placeholder="tell me your name"
+          value={name}
+          onChange={event => setName(event.target.value)}>
+        </input>
+      </label>}
+      {currentQuestion === 2 && <label>
+      How would you describe the trouble you're having {name}, duck-friend?
+        <select value={problemType} onChange={event => setProblemType(event.target.value)}>
+          <option value="big bad bug">it's a big bad bug 🐛</option>
+          <option value="don't know how to start">I just don't know how to start</option>
+          <option value="concept confusion">I don't understand a concept I'm working with</option>
+          <option value="coder's block">coder's block: I need ideas for what to build and I don't have them</option>
```

Comment:
need to edit values so that they can be effortlessly interpolated as nouns: 
"OK, {name} with the {problemType}" 
===> "OK, buddy with the big bad bug" 👍
===> "OK, buddy with the don't know how to start" 👎